pkg/cli/admin/inspect/admission_webhooks.go

```diff
@@ -0,0 +1,118 @@
+package inspect
+
+import (
+	"fmt"
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	"k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/cli-runtime/pkg/resource"
+)
+
+type mutatingWebhookConfigList struct {
+	*admissionregistrationv1.MutatingWebhookConfigurationList
+}
+
+func (c *mutatingWebhookConfigList) addItem(obj interface{}) error {
```

Comment:
we appear to lack such a function in upstream kube.  I bet it's the typing problem